src/api-service/__app__/onefuzzlib/workers/nodes.py

```diff
@@ -367,8 +367,8 @@ def is_outdated(self) -> bool:
     def is_too_old(self) -> bool:
         return (
             self.scaleset_id is not None
```

Comment:
Not for this PR, but presumably this hasn't actually been (logically) optional for quite some time.